TrackingBackend/app/tracker.py

```diff
@@ -1,36 +1,44 @@
-from __future__ import annotations
-from .logger import get_logger
-from .config import TrackerConfig
-from app.processes import EyeProcessor, Camera
-from multiprocessing.managers import SyncManager
-from app.utils import clear_queue
 from queue import Queue
 from .types import EyeData
-import cv2
+from cv2.typing import MatLike
+from .logger import get_logger
+from app.utils import clear_queue
+from .config import EyeTrackConfig
```

Comment:
another little nitpick, I've noticed that we sometimes import by `app.module.thing` and sometimes by `.local_file`, I'd say let's stick with `from app.module` style, it's esier to follow and for tools to correct :D

Reply:
Will definitely change this!